### ISSUE TITLE
File fetching error when using Flowershow Cloud

### ISSUE THREAD

**Issue by anuveyatsu** [datopian/flowershow#596]
If you go here https://hiring.flowershow.app/ and click on the link called `/engineering/python` it throws an error.

Logs that I found:

```
'Could not read datopian/hiring/engineering/python-developer.md.md or datopian/hiring/engineering/python-developer.md/index.md on branch main from GitHub: Error: Could not read datopian/hiring/engineering/python-developer.md/index.md from GitHub: Error: Failed to fetch GitHub file: Not Found'
```

which indicates that Flowershow backend has some additional logic appending `.md` to the path or trying to search for `index.md` in the directory.

My raw markdown is very basic - https://github.com/datopian/hiring - that just links to other markdown files within the repo. I use `/engineering/python-developer.md` as relative path but Github itself resolves it to https://github.com/datopian/hiring/blob/main/engineering/python-developer.md i.e., by adding `blob/main` in between.

**Comment by rufuspollock** [comment]
@anuveyatsu good point and we'll check how we are computing the path to look this up.

**Comment by rufuspollock** [comment]
@anuveyatsu ok this is because our fetch logic adds `.md` automatically and we should not add in the case where the file link already had .md.

@olayway you reckon this is as easy fix so let's do that.
